docs/core-concepts/rid-catalog.md

```diff
@@ -0,0 +1,111 @@
+# Runtime IDentifier (RID) catalog
+
+## What are RIDs?
+RID is short for *Runtime IDentifier*.RIDs are used to identify target operating systems where an application or asset 
+(i.e. assembly) will run. They look like this: "ubuntu.14.04-x64", "win7-x64", "osx.10.11-x64". 
+For the packages with native dependencies, as we saw, it will designate on which platforms the package can be restored. 
+
+It is important to note that RIDs are really opaque strings. This means that they have to match exactly for operations 
+using them to work. As an example, let us consider the case of [Elementary OS](https://elementary.io/), which is a straightforward clone of 
```

Comment:
"This means that they have to match exactly for operations using them to work." 
I think I understand what you're trying to say here but I don't understand this sentence.